src/SprykerEco/Yves/Ratepay/Theme/default/views/summary/summary.twig

```diff
@@ -0,0 +1,50 @@
+    quoteTransfer: required
+} %}
+
+{% block body %}
+    {% if data.quoteTransfer.payment.paymentMethod == 'INVOICE' %}
+        {% set ratepayPaymentTransfer = data.quoteTransfer.payment.ratepayInvoice %}
+    {% elseif data.quoteTransfer.payment.paymentMethod == 'ELV' %}
+        {% set ratepayPaymentTransfer = data.quoteTransfer.payment.ratepayElv %}
+        <tr>
+            <td>{{ 'payment.ratepay.bank_account' | trans }}:</td>
+            <td>
+                <div>{{ 'payment.ratepay.bank_account.holder' | trans }} - {{ ratepayPaymentTransfer.bankAccountHolder }}</div>
+                <div>{{ 'payment.ratepay.bank_account.iban' | trans }} - {{ ratepayPaymentTransfer.bankAccountIban }}</div>
+                <div>{{ 'payment.ratepay.bank_account.bic' | trans }} - {{ ratepayPaymentTransfer.bankAccountBic }}</div>
+            </td>
+        </tr>
+    {% elseif data.quoteTransfer.payment.paymentMethod == 'PREPAYMENT' %}
+        {% set ratepayPaymentTransfer = data.quoteTransfer.payment.ratepayPrepayment %}
+    {% elseif data.quoteTransfer.payment.paymentMethod == 'INSTALLMENT' %}
+        {% set ratepayPaymentTransfer = data.quoteTransfer.payment.ratepayInstallment %}
+        <tr>
+            <td>{{ 'summary.ratepay.installment.plan_details' | trans }}:</td>
+            <td>
+                <div>{{ 'summary.ratepay.installment.type' | trans }} - {{ ratepayPaymentTransfer.installmentCalculationType }}</div>
+                <div>{{ 'summary.ratepay.installment.rates_number' | trans }} - {{ ratepayPaymentTransfer.installmentNumberRates|number_format }}</div>
+                <div>{{ 'summary.ratepay.installment.amount_per_month' | trans }} - {{ ratepayPaymentTransfer.installmentRate|money }}</div>
+                <div>{{ 'summary.ratepay.installment.last_month_amount' | trans }} - {{ ratepayPaymentTransfer.installmentLastRate|money }}</div>
+                <div>{{ 'summary.ratepay.installment.annual_percentage_rate' | trans }} - {{ ratepayPaymentTransfer.installmentAnnualPercentageRate|number_format(2, ',') }} &#37;</div>
+                <div>{{ 'summary.ratepay.installment.service_charge' | trans }} - {{ ratepayPaymentTransfer.installmentServiceCharge|money }}</div>
+                <div>{{ 'summary.ratepay.installment.interest_amount' | trans }}- {{ ratepayPaymentTransfer.installmentInterestAmount|money }}</div>
+                <div>{{ 'summary.ratepay.installment.order_grand_total' | trans }} - {{ quoteTransfer.getTotals.getGrandTotal|money }}</div>
+                <div>{{ 'summary.ratepay.installment.plan_grand_total' | trans }}  - {{ ratepayPaymentTransfer.installmentGrandTotalAmount|money }}</div>
+            </td>
+        </tr>
+    {% endif %}
+    {% if ratepayPaymentTransfer is defined %}
+        {% block javascripts %}
+            <script language="JavaScript">
+                var di = {t:'{{ ratepayPaymentTransfer.deviceFingerprint }}',v:'{{ ratepayPaymentTransfer.deviceIdentSId }}',l:'Checkout'}; </script>
```

Comment:
Is it possible do not use global variable?

Reply:
No, because external library refers to that variable